packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json

```
@@ -0,0 +1,12 @@
+{
+  "pci_projects_project_kubernetes_details_service_add_oidc_provider_title": "Configurer le fournisseur OIDC",
+  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OpenID Connect comme Dex, Keycloack ou un fournisseur SaaS, à la condition que soient supportés la fonctionnalité \"OpenID Connect discovery\", le TLS et un certificat CA signé.",
```

```suggestion
  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OpenID Connect comme Dex, Keycloak ou un fournisseur SaaS, à la condition que soient supportés la fonctionnalité \"OpenID Connect discovery\", le TLS et un certificat CA signé.",
```

Relates to: DTRSD-58923